x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/trusted_apps_translations.ts

```diff
@@ -135,18 +135,18 @@ export const POLICY_ARTIFACT_TRUSTED_APPS_LABELS = Object.freeze({
     }
   ),
   layoutTitle: i18n.translate('xpack.securitySolution.endpoint.policy.trustedApps.layout.title', {
-    defaultMessage: 'Assigned trusted apps',
+    defaultMessage: 'Assigned trusted applications',
   }),
   layoutAssignButtonTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.trustedApps.layout.assignToPolicy',
     {
-      defaultMessage: 'Assign trusted apps to policy',
+      defaultMessage: 'Assign trusted applications to policy',
     }
   ),
   layoutViewAllLinkMessage: i18n.translate(
     'xpack.securitySolution.endpoint.policy.trustedApps.layout.about.viewAllLinkLabel',
     {
-      defaultMessage: 'view all trusted apps',
+      defaultMessage: 'view all trusted applications',
```

Comment:
Same with this? Does the 'v' need to be capitalized?

Reply:
Same for blocklist, but thanks for pointing this out!